.github/workflows/test.action.yml

```diff
@@ -26,4 +26,25 @@ jobs:
           node-version: ${{ matrix.node-version }}
       - run: yarn install
       - run: yarn build
-      - run: yarn test
\ No newline at end of file
+      - run: yarn test
+  
+  cypress-run:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
+
+    steps:
+     - name: Checkout
+       uses: actions/checkout@v2
+       
+     - name: Use Node.js ${{ matrix.node-version }}
+       uses: actions/setup-node@v2
+       with:
+         node-version: ${{ matrix.node-version }}
+
+     - name: Cypress run
+       uses: cypress-io/github-action@v2
```

Comment:
Also, don't forget to always add new line at the end, GitHub will show this as this red outlined minus sign.